File: lab_2/generate/generator.java
import java.util.Random;

public class Main {
    public static void main(String[] args) {
        Random rand = new Random();
        StringBuilder sequence = new StringBuilder();

        for (int i = 0; i < 128; i++) {
            int bit = rand.nextInt(2); 
            sequence.append(bit);
        }

        System.out.println("Length: " + sequence.length());
        System.out.println(sequence.toString());
    }
}

File: lab_2/generate/generator.py
# -*- coding: utf-8 -*-

import random


def main():
    """Main generator function"""
    sequence: str = ""
    
    for _ in range(128):
        bit: int = random.randint(0,1)
        sequence += str(bit)

    print(sequence)


if __name__ == "__main__":
    main()
